commentManager.js

```diff
@@ -200,3 +200,40 @@ exports.changeAcceptedState = function(padId, commentId, state, callback){
     callback(null, commentId, comment);
   });
 }
+
+exports.changeCommentText = function(padId, commentId, commentText, callback){
+  // Given a comment we update the comment text
+  // We need to change readOnly PadIds to Normal PadIds
+  var isReadOnly = padId.indexOf("r.") === 0;
+  if(isReadOnly){
+    readOnlyManager.getPadId(padId, function(err, rwPadId){
+      padId = rwPadId;
+    });
+  };
+
+  // If we're dealing with comment replies we need to a different query
+  var prefix = "comments:";
+  if(commentId.substring(0,7) === "c-reply"){
+    prefix = "comment-replies:";
+  }
+
+  var commentTextIsEmpty = commentText.length > 0;
+  if(commentTextIsEmpty){
+    //get the entry
+    db.get(prefix + padId, function(err, comments){
+      if(ERR(err, callback)) return;
+
+      //add the entry for this pad
+      var comment = comments[commentId];
+      comment.text = commentText;
+      comments[commentId] = comment;
+
+      //save the new element back
+      db.set(prefix + padId, comments);
```

Comment:
`//add the entry for this pad`: we're not adding, but updating the entry, right?

`//save the new element back`: it's not a new element